app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -112,6 +112,11 @@ def index_relation_environment(query)
     param_group :repo
     def create
       repo_params = repository_params
+      unless RepositoryTypeManager.creatable_repository_types.keys.include?(repo_params[:content_type])
```

Comment:
Why not use https://github.com/Katello/katello/pull/5570/files#diff-f72e23b04b55d404ace586486635dc13R31 here?